src/__main__.py

```diff
@@ -1,18 +1,34 @@
 from pathlib import Path
+from pprint import pprint as pp
 import sys
 from .parse_pst import get_email
 
 
 def email_digest(pst_file): 
-    '''function to scan target pst every hour and output a digest of recent activity'''
-    test = get_email(pst_file)
-    print(test)
+    emails = get_email(pst_file)
+    FIELD = 'subject'
+    digest = []
+    for mail in emails:
+        for key in mail:
+            if key in FIELD:
+                message = {}
+                message['sender'] = mail['sender']
+                message['subject'] = mail['subject']
+                message['delivery_time'] = mail['delivery_time']
+                digest.append(message)
+                    
+  
+    for message in digest:
+        result = ' | '.join(map(str, message.values()))
+        print(result)
+    print(len(digest))
 
 
 if __name__ == "__main__":
     if len(sys.argv) == 2:
-        pst_file = Path(sys.argv[1])
+        pst_file = str(Path(sys.argv[1]))
```

Comment:
as per Slack:

```
>>> from pathlib import Path
>>> file = Path("/Users/bbelderbos/Downloads") / "todos"
>>> file
PosixPath('/Users/bbelderbos/Downloads/todos')

>>> file.resolve()
PosixPath('/Users/bbelderbos/Dropbox/todos')

# nope, google -> https://stackoverflow.com/questions/44315815/python-pathlib-path-object-not-converting-to-string

>>> file.as_posix()
'/Users/bbelderbos/Downloads/todos'
```

Reply:
Without converting the posix path to a string, I get a TypeError unsupported string object type. Don't I want to convert the posix path to a str so that it can be passed to `get_email()`?

Reply:
Yep reproduced: 

```
TypeError: pypff_file_open: unsupported string object type.
```

So in that case I don't think it's worth it to create a `Path` object in the first place. `pst_file = sys.argv[1]` works fine for me.